ovn_k8s/processor/policy_processor.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2016 Nicira, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ovs import vlog
+
+import ovn_k8s.processor as processor
+import ovn_k8s.watcher.policy_watcher
+
+log = vlog.Vlog('policy_processor')
+pw = ovn_k8s.watcher.policy_watcher
+
+
+class PolicyProcessor(processor.BaseProcessor):
+
+    def __init__(self, pool):
+        super(PolicyProcessor, self).__init__()
+        self._pool = pool
+        self._np_watcher_threads = {}
+
+    def _process_ns_event(self, event):
+        namespace = event.source
+        # TODO(salv-orlando): handle namespace isolation change events
+        if event.event_type == 'ADDED':
+            log.dbg("Namespace %s added - spawning policy watcher" % namespace)
+            watcher_thread = pw.create_namespace_watcher(
+                namespace, self._pool)
+            self._np_watcher_threads[namespace] = watcher_thread
+        elif event.event_type == 'DELETED':
+            watcher_thread = self._np_watcher_threads.pop(namespace)
+            pw.remove_namespace_watcher(watcher_thread)
+
+    def process_events(self, events):
+        log.dbg("Processing %d events from queue" % len(events))
+        for event in events[:]:
```

Comment:
If we add a comment here about where these different events are coming from, it makes it  easy. For e.g namespace events come from namespace watcher.